news/CHANGES.md

```diff
@@ -1,3 +1,29 @@
+### v0.1.15:
+
+[full changelog](https://github.com/pymedusa/Medusa/commits/master)
+
+**New Features**:
+- Caching for the IMDB and Trakt recommended lists. First time load might take long. Subsequent load will load much faster
```

Comment:
The first time we download the lists it may take a while but every time after that it'll load faster.